components/__tests__/SingleReport.test.js

```diff
@@ -0,0 +1,51 @@
+import { mount, flushPromises } from "@vue/test-utils";
```

Comment:
**suggestion:** This is the file where I might move the switch tests and then check that they appear for each report, show the label, show the default true one, and emit the expected event on click.

Reply:
Question does vue test utils render that far? Cause I keep getting false for the component existing

Reply:
🙌